signer/tests/integration/transaction_coordinator.rs

```diff
@@ -3422,6 +3426,428 @@ async fn test_conservative_initial_sbtc_limits() {
+        max_fee: 100_000,
+        txid: StacksTxId::from([123; 32]),
+        sender_address: PrincipalData::from(StandardPrincipalData::transient()).into(),
+    };
+    for (_, db, _, _) in signers.iter() {
+        db.write_withdrawal_request(&withdrawal_request)
+            .await
+            .unwrap();
+    }
+
+    // =========================================================================
+    // Step 7 - Confirm WITHDRAWAL_MIN_CONFIRMATIONS more blocks so that
+    //          the signers process the withdrawal request.
+    // -------------------------------------------------------------------------
+    // - Generate WITHDRAWAL_MIN_CONFIRMATIONS blocks the deposit request.
+    //   This will trigger the block observer to update the database.
+    // - Each RequestDecider process should vote on the withdrawal request
+    //   and submit the votes to each other.
+    // - The coordinator should submit a sweep transaction with an output
+    //   fulfilling the withdrawal request.
+    // =========================================================================
+    for _ in 0..WITHDRAWAL_MIN_CONFIRMATIONS {
+        faucet.generate_blocks(1);
+        wait_for_signers(&signers).await;
+    }
+
+    let (ctx, _, _, _) = signers.first().unwrap();
+    let mut txids = ctx.bitcoin_client.inner_client().get_raw_mempool().unwrap();
+    assert_eq!(txids.len(), 1);
+
+    let block_hash = faucet.generate_blocks(1).pop().unwrap();
+
+    wait_for_signers(&signers).await;
+
+    // =========================================================================
+    // Step 8 - Assertions
+    // -------------------------------------------------------------------------
+    // - The first transactions should be a rotate keys contract call. And
+    //   because of how we set up our mocked stacks client, each
+    //   coordinator submits a rotate keys transaction before they do
+    //   anything else.
+    // - The last transaction should be us accepting the withdrawal request
+    //   and burning sBTC.
+    // - Is the sweep transaction in our database.
+    // - Does the sweep outputs to the right scriptPubKey with the right
+    //   amount.
+    // =========================================================================
+    let sleep_fut = tokio::time::sleep(Duration::from_secs(5));
+    let broadcast_stacks_txs: Vec<StacksTransaction> = stacks_tx_stream
+        .take_until(sleep_fut)
+        .collect::<Vec<_>>()
+        .await
+        .into_iter()
+        .collect::<Result<Vec<_>, _>>()
+        .unwrap();
+
+    more_asserts::assert_ge!(broadcast_stacks_txs.len(), 2);
+    // Check that the first N - 1 are all rotate keys contract calls.
+    let rotate_keys_count = broadcast_stacks_txs.len() - 1;
+    for tx in broadcast_stacks_txs.iter().take(rotate_keys_count) {
+        assert_stacks_transaction_kind::<RotateKeysV1>(tx);
+    }
+    // Check that the Nth transaction is the complete-deposit contract
```

Comment:
```suggestion
    // Check that the Nth transaction is the accept-withdrawal contract
```

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/1403/commits/3105605f75746d0eaba7a86215daaf8142683f4d